ynr/apps/bulk_adding/views/parties.py

```diff
@@ -228,6 +228,8 @@ def form_valid(self, formsets):
                         person = Person.objects.get(
                             pk=int(data["select_person"])
                         )
+
+                    # TODO check about updating PartyDescription here
```

Comment:
If this is an open question - then yes, it would be good to update the description here.

The reason is that this person's candidacy might have been made some time before we knew the description - the membership is valid, but it's likely we don't know the description. Because of that, we want to take the SOPN version at this stage.

Reply:
We investigated this but came across a bug looking up the Party object in this view after a user has clicked the "Load more parties" option. But in the interests of not blocking other PR's we thought it best to tackle this (and then storing the PartyDescription) separately, so I'll add an issue. As it stands, the `party_name` will be stored against a Person due to the other changes in to the helper we made.

Reply:
Good call - this is something that I've changed a bit anyway, so the chances are working on this would have caused more problems!